Makefile

```diff
@@ -75,16 +75,22 @@ manifests: controller-gen
 
 unit-test:
 	go clean -testcache
-	go list ./... | grep -v e2e | xargs go test -v -race
+	go list ./... | grep -v e2e | xargs go test -p 8 -v -race
 
 e2e:
 	./hack/run-e2e-kind.sh
 
-e2e-test-scheduling:
-	E2E_TYPE=SCHEDULING ./hack/run-e2e-kind.sh
+e2e-test-schedulingbase:
+	E2E_TYPE=SCHEDULINGBASE ./hack/run-e2e-kind.sh
 
-e2e-test-job:
-	E2E_TYPE=JOB ./hack/run-e2e-kind.sh
+e2e-test-schedulingaction:
```

Comment:
What's the diff between schedulingbase and schedulingaction

Reply:
schedulingbase is the basic scheduling test, schedulingaction is just include action plugin.